tools/sggolines/tools.go

```diff
@@ -10,13 +10,28 @@ import (
 
 	"go.einride.tech/sage/sg"
 	"go.einride.tech/sage/sgtool"
+	"go.einride.tech/sage/tools/sggofumpt"
 )
 
 const (
 	name    = "golines"
 	version = "0.12.2"
 )
 
+// Run golines on all Go files in the current git root with gofumpt as default formatter.
+func Run(ctx context.Context) error {
+	sg.Deps(ctx, sggofumpt.PrepareCommand)
+	return Command(
+		ctx,
+		"--base-formatter=gofumpt",
```

Comment:
Hm, maybe relevant: https://github.com/segmentio/golines/issues/100
Didn't have time to read it all through. I just remember seeing this ...

Reply:
We can always run with what you've done now - and just keep this issue in mind, in case we see something like that later.

Reply:
Ohh nice find! I'll update